homeassistant/components/light/wemo.py

```diff
@@ -105,6 +105,7 @@ def turn_on(self, **kwargs):
         elif ATTR_RGB_COLOR in kwargs:
             xycolor = color_util.color_RGB_to_xy(
                 *(int(val) for val in kwargs[ATTR_RGB_COLOR]))
+            self.device.turn_on(level=xycolor[2])
```

Comment:
Not sure if I like this because it will cause the light to flicker if we call turn_on with both ATTR_RGB_COLOR and ATTR_BRIGHTNESS. Maybe use `kwargs.setdefault(ATTR_BRIGHTNESS, xycolor[2])` here?